tests/test_report.py

```diff
@@ -11,6 +11,32 @@ def create_report(report_type, report_start_date, report_end_date):
     )
 
 
+@pytest.mark.vcr()
+def test_report_create_with_additional_columns(report_start_date, report_end_date):
+    report = easypost.Report.create(
+        type="shipment",
+        start_date=report_start_date,
+        end_date=report_end_date,
+        additional_columns=["from_name", "from_company"],
+    )
+
+    assert isinstance(report, easypost.Report)
+    assert str.startswith(report.id, "shprep_")
+
+
+@pytest.mark.vcr()
+def test_report_create_with_columns(report_start_date, report_end_date):
+    report = easypost.Report.create(
+        type="shipment",
+        start_date=report_start_date,
+        end_date=report_end_date,
+        columns=["usps_zone"],
+    )
+
+    assert isinstance(report, easypost.Report)
+    assert str.startswith(report.id, "shprep_")
```

Comment:
We don't need the ID check

Reply:
I believe from the discussion yesterday, we are going to have these two assertions, similar to other unit tests for `report`.

Reply:
Not exactly, we don't need to assert the ID since all we are testing is if the report gets created correctly. We are testing the ID elsewhere. We should assert that what comes back is a report (indicating success with the params passed) and then leaving a comment similar to what we do with batches saying we can't test this more in-depth because the report gets queued for generation and most likely isn't available quick enough for us to retrieve.

Reply:
fixed.